### ISSUE TITLE
Web UI: Status bar in the wrong place and not clickable on FreeBSD

### ISSUE THREAD

**Issue by yzgyyang** [MediaBrowser/Emby#3482]
![image](https://user-images.githubusercontent.com/16748524/48972527-15b75900-effa-11e8-8d81-42b6ca38a3a7.png)

On FreeBSD 11.2 with the latest release of emby-server, I encountered this problem on all videos. The progress bar is at the top of the screen and unclickable. I cannot reproduce this on my Ubuntu-backed emby-server.

**Comment by LukePulverenti** [comment]
What version of Emby Server?

**Comment by yzgyyang** [comment]
@LukePulverenti I just realized that I installed emby from pkg **stable**, so it's v3.5.2.0 instead of the latest one. I will try the actual latest release and report back in a while.

```
➜  ~ pkg info emby-server
emby-server-3.5.2.0
Name           : emby-server
Version        : 3.5.2.0
Installed on   : Sat Nov 24 12:09:03 2018 CST
Origin         : multimedia/emby-server
Architecture   : FreeBSD:11:*
Prefix         : /usr/local
Categories     : multimedia
Licenses       : GPLv2
Maintainer     : woodsb02@FreeBSD.org
WWW            : https://emby.media/
Comment        : Home media server built using mono and other open source technologies
Options        :
	X11            : on
Annotations    :
	repo_type      : binary
	repository     : FreeBSD
Flat size      : 44.2MiB
Description    :
Emby Server is a home media server built on top of other popular
open source technologies such as Service Stack, jQuery, jQuery mobile,
and Mono.

It features a REST-based api with built-in documention to facilitate
client development. We also have client libraries for our api to enable
rapid development.

WWW: https://emby.media/
```

**Comment by yzgyyang** [comment]
@LukePulverenti I can confirm that the latest release v3.5.3.0 fixed this UI problem (among some others). Sorry for the noice!

**Comment by LukePulverenti** [comment]
Thanks for the feedback.

**Comment by kitten77** [comment]
I had the same problem from the quarterly package, had some issues when I changed from quarterly to latest. 
Not really sure what i encountered, but for some reason the ffmpeg didn't play any of my files with release v3.5.3.0.
This after a pkg update & upgrade on latest branch.
 @yzgyyang you did a pkg update & upgrade after you changed to the latest in  /etc/pkg/FreeBSD.conf
  url: "pkg+http://pkg.FreeBSD.org/${ABI}/latest" ?

**Comment by yzgyyang** [comment]
@kitten77 emby is currently built against quarterly, and the latest is broken (I'm experiencing this as well). There's an ongoing bug report about this, and there seem to be no simple fixes but to wait for the maintainer. https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=232966

**Comment by LukePulverenti** [comment]
This is resolved in our beta packages and will also be included in the upcoming 3.6 stable. Thanks guys.
